deepmd/descriptor/se_r.py

```diff
@@ -475,6 +471,33 @@ def _pass_filter(self,
             inputs_i = inputs
             inputs_i = tf.reshape(inputs_i, [-1, self.ndescrpt])
             type_i = -1
+            if len(self.exclude_types):
```

Comment:
This branch is repeating with `se_a`, except some shape difference. Write a method in the base class descriptor and reuse the method in both `se_r` and `se_a`?

Reply:
Merged.